modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -54,15 +55,16 @@ exports.update = function (req, res) {
 exports.changeProfilePicture = function (req, res) {
   var user = req.user;
   var message = null;
+  var upload = multer({ dest: './modules/users/client/img/profile/uploads/' }).single('newProfilePicture');
 
   if (user) {
-    fs.writeFile('./modules/users/client/img/profile/uploads/' + req.files.file.name, req.files.file.buffer, function (uploadError) {
-      if (uploadError) {
+    upload(req, res, function (err){
+      if(err) {
         return res.status(400).send({
           message: 'Error occurred while uploading profile picture'
         });
       } else {
-        user.profileImageURL = 'modules/users/client/img/profile/uploads/' + req.files.file.name;
+        user.profileImageURL = 'modules/users/client/img/profile/uploads/' + req.file.filename;
```

Comment:
See my note above.
@mleanos is correct that
```javascript
user.profileImageURL = req.file.path 
```

is not correct. You should rather include the full path and not rely on the req.file.path variable for the location of the image file. Use the config variable as described above in my comment.